AsyncDisplayKit/ASTableView.mm

```diff
@@ -849,14 +858,15 @@ - (void)tableViewCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath *)indexPa
   [self beginUpdates];
   ASCellNode *node = [_dataController nodeAtIndexPath:indexPath];
   
-  CGSize constrainedSize = [self dataController:_dataController constrainedSizeForNodeAtIndexPath:indexPath];
+  ASSizeRange constrainedSize = [self dataController:_dataController constrainedSizeForNodeAtIndexPath:indexPath];
   if (state != UITableViewCellStateDefaultMask) {
     // Edit control or delete confirmation was shown and size of content view was changed.
     // The new size should be taken into consideration.
-    constrainedSize.width = MIN(cell.contentView.frame.size.width, constrainedSize.width);
+    constrainedSize.min.width = MIN(cell.contentView.frame.size.width, constrainedSize.min.width);
+    constrainedSize.max.width = MIN(cell.contentView.frame.size.width, constrainedSize.max.width);
```

Comment:
Oh and it's worth to mention that both min and max sizes must be resolve here. I'm pretty about the math, but I might be wrong!